src/core/ext/xds/xds_client.cc

```diff
@@ -1477,6 +1477,10 @@ XdsClient::XdsClient(
     gpr_log(GPR_INFO, "[xds_client %p] creating xds client", this);
   }
   GPR_ASSERT(bootstrap_ != nullptr);
+  if (bootstrap_->node()) {
+    gpr_log(GPR_INFO, "[xds_client %p] creating xds client with node ID: %s",
```

Comment:
I think this should just say `xDS node ID: %s`.

Reply:
Done.